src/actions/receive-profile.js

```diff
@@ -966,30 +965,6 @@ export async function doSymbolicateProfile(
   dispatch(doneSymbolicating());
 }
 
-class TimeoutError extends Error {
-  name = 'TimeoutError';
-}
-
-function makeTimeoutRejectionPromise(durationInMs) {
-  return new Promise((_resolve, reject) => {
-    setTimeout(() => {
-      reject(new TimeoutError(`Timed out after ${durationInMs}ms`));
-    }, durationInMs);
-  });
-}
-
-export async function checkIfWebChannelUsableForSymbolication(): Promise<boolean> {
-  try {
-    return await Promise.race([
-      querySupportsGetProfileAndSymbolicationViaWebChannel(),
-      makeTimeoutRejectionPromise(5000),
-    ]);
-  } catch (e) {
-    // It timed out or some other error happened. Don't use the WebChannel.
-    return false;
-  }
-}
-
```

Comment:
This was a left-over from the move of this logic to BrowserConnection and was never removed here.

Reply:
Thanks for noticing that.